### ISSUE TITLE
Native code compile failed when installing

### ISSUE THREAD

**Issue by alanpog** [aws/awsmobile-cli#43]
On a macOS 10.13.2, `npm install -g awsmobile-cli`:
```
[execsync v1.0.2] Attempting to compile native extensions.
[execSync v1.0.2]
    Native code compile failed!!
+ awsmobile-cli@1.0.7
```

**Comment by UnleashedMind** [comment]
this is probably because you are using an earlier version of node. 
check this post:
https://stackoverflow.com/questions/39812448/angular-cli-installation-failure

please execute 
node -v 
and let us know your node version

If you upgrade your node to be above 8.0.0, it should resolve this issue.

**Comment by alanpog** [comment]
I'm already on `9.4.0`, which I believe is the latest.
```
AlanMBP:p2 alanp$ node -v
v9.4.0
```

**Comment by UnleashedMind** [comment]
Thanks, we've found that the issue was caused by an older version of a dependency package. 
This issue should now be resolved by the latest release 1.0.8
Please verify.

**Comment by alanpog** [comment]
Yes, it works now. Thanks!
